packages/expo-modules-autolinking/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Maintain hierarchical order when linking isolated modules ([#24351](https://github.com/expo/expo/pull/24351) by [@byCedric](https://github.com/byCedric))
```

Comment:
would be good to mention more about that is related to pnpm support.

Reply:
I think we should emphasize this more in the blogpost. The reason for that is because it's not _just_ pnpm. This fix also enables `npm install --install-strategy=linked` [which is available already](https://github.com/npm/cli/pull/6078) - but still experimental.

Reply:
that makes sense 👍